### DISCUSSION TITLE
How to use unocss reset with vite-lit?

### DISCUSSION THREAD

**Discussion by vasishath** [unocss/unocss#2753] [Q&A]
Hi, I was wondering where I'd put my CSS resets when using lit with shadowcss mode, when wanting to use @unocss/reset. I tried adding injectReset to vite.config but it doesn't work.

**Comment by serkodev** [comment]
If possible, could you please share your source code for reproduction of the issue?
This would make it much easier for everyone to understand the details of your issue.

**Reply by vasishath** [reply]
```javascript
import { html, css } from "lit";
import { customElement, property } from "lit/decorators.js";



@customElement("vk-col")
export class VKCol extends LitElement {
static styles = css`@unocss-placeholder`;
  @property()
  class?: string = "";

  render() {
    return html`
      <div part="class" class="flex flex-col justify-center items-center">
      <slot></slot>
      </div>
    `;
  }
}
```

I wish to add tailwindreset.css to this but direct import of css file is deprecated in vite.

**Comment by McNerdius** [comment]
what i've been doing: 

``` typescript
import { LitElement, unsafeCSS } from 'lit';
import preflight from '@unocss/reset/tailwind.css?inline';

@customElement('foo-element')
export class FooElement extends LitElement {
    static styles = [ unsafeCSS(preflight), css`@unocss-placeholder` ];
}
```

Only a few days into this though.

**Reply by vasishath** [reply]
Hmm but this way, we have to include the preflight into every component. I wish injectReset worked here
